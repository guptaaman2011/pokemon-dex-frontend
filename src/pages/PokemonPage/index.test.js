import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import { PokemonPage } from "..";

describe("PokemonPage", () => {
  it("should render without breaking", () => {
    const wrapper = shallow(<PokemonPage />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
