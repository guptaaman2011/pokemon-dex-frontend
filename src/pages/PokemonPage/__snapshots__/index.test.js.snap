// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonPage should render without breaking 1`] = `
<div>
  <PokemonHeader />
  <Row>
    <Col
      span={12}
    >
      <Pagination
        defaultPageSize={20}
        onChange={[Function]}
        total=""
      />
    </Col>
    <Col
      span={6}
    >
      Search Pokemon
    </Col>
    <Col
      span={6}
    >
      Sort By
    </Col>
  </Row>
  <hr />
  <List
    dataSource={Array []}
    grid={
      Object {
        "gutter": 16,
        "lg": 4,
        "md": 4,
        "sm": 2,
        "xl": 5,
        "xs": 1,
        "xxl": 3,
      }
    }
    renderItem={[Function]}
  />
  <div>
    <Pagination
      defaultPageSize={20}
      onChange={[Function]}
      total=""
    />
  </div>
  <br />
</div>
`;
